app/core/events_parser.py

```diff
@@ -34,7 +34,6 @@ def parse_premis_events(input_xml: bytes):
     events = [
         event
         for event in (parse_premis_event(element) for element in elements)
```

Comment:
Since we no longer check if the event is archived and/or valid this statement can be simplified to a normal list comprehension without the generator.

`events = [parse_premis_event(element)] for element in elements` should work. Now the `event for event` part is useless as there is no `if` anymore :)